powerbi-docs/visuals/power-bi-visualization-export-data.md

```diff
@@ -74,6 +74,8 @@ To follow along, open the [Procurement analysis sample report](../sample-procure
     ![Screenshot of the Export data screenshot with the Summarized data, xlsx, and Export options called out.](media/power-bi-visualization-export-data/power-bi-export-data5.png)
 
     If you applied filters to the visualization, the exported data will export as filtered. When you select  **Export**, your browser prompts you to save the file. Once saved, open the file in Excel.
+    
+    All the data used by the hierarchy is exported, not simply the data used for the current drill level for the visual. For example, if you've drilled down to another level of the visualization, the exported data will include all of the data in the hierarchy, not just the data used to create the visual at its currently drilled level.
```

Comment:
```suggestion
    All the data used by the hierarchy is exported, not simply the data used for the current drill level for the visual. For example, if the visualization has not been drilled down yet from the top level, the exported data will include all of the data in the hierarchy, not just the data used to create the visual at its currently drilled level.
```

Reply:
Suggesting the change above because it sounds like you have to start drilling for this behavior to occur. Instead, the point is that any drill state of the visual will have no effect on the exported data other than the slicer selections.